lua/neogit/popups/rebase/actions.lua

```diff
@@ -0,0 +1,71 @@
+local a = require("plenary.async")
+local git = require("neogit.lib.git")
+local input = require("neogit.lib.input")
+local status = require("neogit.status")
+
+local CommitSelectViewBuffer = require("neogit.buffers.commit_select_view")
+local FuzzyFinderBuffer = require("neogit.buffers.fuzzy_finder")
+
+local M = {}
+
+function M.base_branch()
+  return git.config.get("neogit.baseBranch").value or "master"
+end
+
+function M.onto_base(popup)
+  git.rebase.rebase_onto(M.base_branch(), popup:get_arguments())
+  a.util.scheduler()
+  status.refresh(true, "rebase_master")
+end
+
+-- TODO: Set upstream if unset
+function M.onto_upstream(popup)
+  git.rebase.rebase_onto(git.repo.upstream.remote .. " " .. git.repo.upstream.branch, popup:get_arguments())
```

Comment:
Won't this be treated as one argument `"upstream branch"` rather than `"upstream", "branch"`?

Running rebase locally requires those to be two separate shell arguments, but there may be something I've missed

Reply:
I think it was so I didnt need to change the function signature of `rebase_onto`, since you can also use it without a remote. Maybe its a better idea to pass an opts table with a remote key, and use that if present

Reply:
I think an table is the best here for versitility.

Does the rebase command work at all now with the "combined" argument?

Reply:
After a bit of reading, it seems like git will happily accept the fully-qualified ref name, f.ex `refs/remotes/origin/master` in place of `origin master`. I think that's the simplest way to go about this.

Not sure what you mean by "combined" - could you elaborate?

Reply:
That sounds better, so `branch + "/" + commit` rather than `branch + " " + commit`

Testing the existing state locally with `ru` gives

![image](https://github.com/TimUntersberger/neogit/assets/25723553/cdffffe5-a5f4-41ac-9e61-349ca1a4f3ed)

I'll make a note about it and you or I will come around to fix it later, don't put any pressure on it now